### PR TITLE
Avoid NullPointerException

### PR DESCRIPTION
**Pull request by culmat**
Some JDBC drivers (db2cc 10.1.0.1) in my case yield strange infos. Just ignore them.

### CODE REVIEW

File: src/main/java/com/iciql/TableInspector.java [hunk 1]
```diff
@@ -115,6 +115,7 @@ void read(DatabaseMetaData metaData) throws SQLException {
             while (rs.next()) {
                 IndexInspector info = new IndexInspector(rs);
                 if (info.type.equals(IndexType.UNIQUE)) {
+                    if(info.name == null) continue;
```
Comment (gitblit) [inline src/main/java/com/iciql/TableInspector.java]: The concept of the fix is good but I'm not crazy about the proposed one-liner; that's not my preferred style.

Consider this alternative:
```java
if (info.name != null && info.type.equals(IndexType.UNIQUE)) {
```

Comment (culmat) [inline src/main/java/com/iciql/TableInspector.java]: `info.name` is used below the if in line 126ff.
I would rather even put it one line up - it only happend to me for `IndexType.UNIQUE` but could happen for other types.

Comment (gitblit) [inline src/main/java/com/iciql/TableInspector.java]: Ah.  I didn't look at the whole method.  Agreed, it should move up.

```java
IndexInspector info = new IndexInspector(rs);
if (info.name == null) {
    continue;
}
```

**Comment by culmat** [comment]
I will make a second request

**Pull request closed without merging**